Gemfile

```diff
@@ -43,12 +43,19 @@ gem 'tzinfo-data', platforms: %i[mswin mswin64 mingw x64_mingw jruby]
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', require: false
 
+gem 'rspec-rails', '~> 6.1'
```

Comment:
if it involved with test, please fill it in group :test or with :development. Not include like this